wire-ios-data-model/Tests/Source/ManagedObjectContext/DatabaseMigrationTests+UserClientUniqueness.swift

```diff
@@ -79,98 +79,6 @@ final class DatabaseMigrationTests_UserClientUniqueness: XCTestCase {
         }
     }
 
-    func testMigratingToMessagingStore_2_107_PreventsDuplicateUserClients() throws {
-        let mappingModelURL = bundle.url(forResource: "MappingModel_2.106-2.107", withExtension: "cdm")
-        let mappingModel = try XCTUnwrap(NSMappingModel(contentsOf: mappingModelURL))
-
-        try migrateStore(
-            sourceVersion: "2.106.0",
-            destinationVersion: "2.107.0",
-            mappingModel: mappingModel,
-            preMigrationAction: { context in
-                insertDuplicateClients(with: clientID, in: context)
-                try context.save()
-
-                let clients = try fetchClients(with: clientID, in: context)
-                XCTAssertEqual(clients.count, 2)
-            },
-            postMigrationAction: { context in
-                try context.performGroupedAndWait { [self] context in                // verify it deleted duplicates
-                    var clients = try fetchClients(with: clientID, in: context)
-                    XCTAssertEqual(clients.count, 1)
-
-                    // verify we can't insert duplicates
-                    context.mergePolicy = NSMergePolicy.mergeByPropertyObjectTrump
-                    insertDuplicateClients(with: clientID, in: context)
-                    try context.save()
-
-                    clients = try fetchClients(with: clientID, in: context)
-                    XCTAssertEqual(clients.count, 1)
```

Comment:
I think we should keep a unit test to make sure that behaviour now works "not to insert duplicates" in future. Also the test can be seen as documentation.

Reply:
this is overlapping test as this tested only the MappingModel. The "not to insert duplicates" is covered in test in the same class: DatabaseMigrationTests+UserClientUniqueness.swift:42